src/query/Index.cpp

```diff
@@ -211,6 +211,25 @@ ProjectionMatch Index::findNearestProjection(const Eigen::VectorXd &location, in
   }
 }
 
+ProjectionMatch Index::findCellInterpolation(const Eigen::VectorXd &location, int n)
+{
+  if (_mesh->getDimensions() == 2) {
+    auto matchedTriangles = getClosestTriangles(location, n);
+    for (const auto &match : matchedTriangles) {
+      auto polation = mapping::Polation(location, _mesh->triangles()[match.index]);
+      if (polation.isInterpolation()) {
+        return {polation, 0.0};
```

Comment:
@fsimonis I made this change (return 0 instead of a distance) such that non-zero distances only occur in case of fall-back.

Maybe I should also do the `distanceStatistics(match.distance);` only when this distance is non-zero ? This way the distance statistics would count the number of fall-backs.

Reply:
Well, the distance is 0, so this makes sense.

Good idea, but then you need to rename the `distanceStatistics` to `fallbackStatistics` or similar.